transport/transporters/kafka/client_config.yaml.go

```diff
@@ -53,6 +53,8 @@ func producerConfig(kafkaTLS bool, clusterCA, clientPrivateKey, clientPublicKey
 	// partition if we retry too quickly.
 	config.Producer.Retry.Backoff = 500 * time.Millisecond
 
+	config.Producer.Partitioner = sarama.NewManualPartitioner
```

Comment:
Because I manually set the `Partition` field in the `sarama.ProduceMessage`, I'm using the `NewManualPartitioner`. This partitioner sends messages to the partition specified in the message's `Partition` field.